Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm

```diff
@@ -251,4 +256,29 @@ void saveCookies(NSHTTPCookieStorage *cookieStorage, CompletionHandler<void()>&&
 }
 #endif // HAVE(NW_PROXY_CONFIG)
 
+#if USE(RUNNINGBOARD) && USE(EXTENSIONKIT)
+bool NetworkProcess::aqcuireLockedFileGrant()
+{
+    NSError* error = nil;
+    m_holdingLockedFileGrant = [WKNetworkingProcessExtension.sharedInstance grant:@"com.apple.common" name:@"FinishTaskInterruptable" error:&error];
+    if (m_holdingLockedFileGrant)
+        RELEASE_LOG(Process, "Successfully took assertion on Network process for locked file");
+    else
+        RELEASE_LOG_ERROR(Process, "Unable to take assertion on Network process for locked file, error = %@", error);
+    return !!m_holdingLockedFileGrant;
+}
+
+void NetworkProcess::invalidateGrant()
+{
+    [m_holdingLockedFileGrant invalidateWithError:nil];
```

Comment:
It seems like logging an error if we can't invalidate would be useful, and only an addition line or two.

Reply:
That's a good point, I'll add logging here.

Thanks for reviewing!